tools/hidl/hidlcomp.cpp

```diff
@@ -5357,12 +5360,22 @@ void EspServInfo::write_esp_binding()
     outs("\treturn -1;\n");
     outs("}\n");
 
-    //method ==> getXsdDefinition
+
+    //method ==> getXmlFilename for xsd and wsdl transformations
+    outf("\nint C%sSoapBinding::getXmlFilename(StringBuffer &filename)\n", name_);
+    outs("{\n");
+    outf("\tfilename.append(\"%s\");\n", packagename);
```

Comment:
You could add the .xml here, producing one line:

outf("\tfilename.append(\"%s.xml\");\n", packagename);

Reply:
Reduced to one line